File: fizzbuzz.js
#!/usr/bin/node

// Language:       JavaScript
// Web site:       https://developer.mozilla.org/en-US/docs/JavaScript
// Last tested on: Ubuntu 24.04.2 LTS
// Requires:       apt-get install nodejs

for (i = 1; i <= 100; i ++) {
    console.log(i % 15 == 0 ? "FizzBuzz" :
                i %  3 == 0 ? "Fizz" :
                i %  5 == 0 ? "Buzz" :
                i)
}

File: fizzbuzz.cpp
// Language:       C++
// Web site:       https://www.open-std.org/JTC1/SC22/WG21/
// Last tested on: Ubuntu 22.04.5
// Requires:       The "g++" package is pre-installed

#include <iostream>
int main() {
    for (int i = 1; i <= 100; i ++) {
        if (i % 15 == 0) {
            std::cout << "FizzBuzz\n";
        }
        else if (i % 3 == 0) {
            std::cout << "Fizz\n";
        }
        else if (i % 5 == 0) {
            std::cout << "Buzz\n";
        }
        else {
            std::cout << i << "\n";
        }
    }
}
